acceptance-tests/lib/env.js

```diff
@@ -29,19 +31,15 @@ const getEnvValue = envVariable => {
   );
 };
 
-const setLocalTestOverrides = (overrides = {}) => {
```

Comment:
I removed this because it's not used anywhere and it was adding complexity. We can add this behavior back if/when we actually need it